bentoml/frameworks/detectron.py

```diff
@@ -0,0 +1,83 @@
+import os
+from bentoml.exceptions import MissingDependencyException
+from bentoml.service.artifacts import BentoServiceArtifact
+from bentoml.service.env import BentoServiceEnv
+
+
+class DetectronModelArtifact(BentoServiceArtifact):
```

Comment:
Can you add the DocString(include a sample code) for the DectronModelArtifact class?  It will make it easier for other users to start using this artifact.

You can use the CoreML artifact as an example:
https://github.com/bentoml/BentoML/blob/df114945f4046f112a4dc0b7afe5e8f80c2d58f8/bentoml/frameworks/coreml.py#L12-L61